sdks/js/packages/core/react/components/organization/billing/upcoming-billing-cycle.tsx

```diff
@@ -181,7 +154,7 @@ export const UpcomingBillingCycle = ({
           billingId
         );
         const invoice = resp?.data?.invoice;
-        if (invoice) {
+        if (invoice && invoice.state) {
```

Comment:
When is `invoice.state` present/not present?

Reply:
empty invoice can be `{}`. which will pass the first check